terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-up.ts

```diff
@@ -390,3 +390,48 @@ export function _calculateScaleUpAmount(
 
   return scaleUpAmount;
 }
+
+export async function innerCreateRunnerConfigArgument(
+  runnerTypeName: string,
+  repositoryName: string,
+  repositoryOwner: string,
+  awsRegion: string,
+  metrics: Metrics,
+  ghesUrlHost: string,
+  isOrgRunner: boolean,
+  isEphemeral: boolean,
+  experimentalRunner: boolean,
+  runnersExtraLabels?: string[] | undefined,
+  runnerLabels?: string[] | undefined,
+  runnerGroupName?: string | undefined,
+  installationId?: number | undefined,
+): Promise<string> {
+  const ephemeralArgument = isEphemeral ? '--ephemeral' : '';
+  const labelsArgument = [
+    `AWS:${awsRegion}`,
+    runnerTypeName,
+    ...(experimentalRunner ? ['experimental.ami'] : []),
+    ...(runnersExtraLabels ? runnersExtraLabels : []),
+    ...(runnerLabels ?? []),
+  ].join(',');
+
+  if (isOrgRunner) {
+    /* istanbul ignore next */
```

Comment:
as you are improving a bit of the code, and needs to update tests.

Lets use this opportunity to remove the tag `/* istanbul ignore next */` in anything that is not error handling.

I saw you even add a few to avoid changing / upgrading some tests. Maybe we should now that we're working on improvements and have less time pressure to be a bit more cautions and work to remove some of the *gambiarra* I added :)